pyrealm/demography/flora.py

```diff
@@ -158,6 +162,11 @@ class PlantFunctionalTypeStrict:
     z_max_prop: float = field(init=False)
     """Proportion of stem height at which maximum crown radius is found."""
 
+    p_foliage_for_reproductive_tissue: float
+    """Proportion of foliage used to calculate reproductive tissue."""
+    p_gpp_for_root_exudation: float
+    """Proportion of GPP used to calculate root exudation."""
+
```

Comment:
Minor tweak to the description and some draft math notation for use in the docs.

```suggestion
    p_foliage_for_reproductive_tissue: float
    """Mass of reproductive tissue as a proportion of foliage mass (:math:`p_{rt}`, -)."""
    p_gpp_for_root_exudation: float
    """Allocation to root exudates as a proportion of GPP (:math:`p_{ex}`, -)"""
```